src/app/GitCommands/Settings/AppSettings.cs

```diff
@@ -1374,7 +1374,7 @@ public static bool ShowDiffForAllParents
             set => SetBool("showdiffforallparents", value);
         }
 
-        public static ISetting<bool> ShowSearchCommit { get; } = Setting.Create(AppearanceSettingsPath, nameof(ShowSearchCommit), false);
+        public static ISetting<bool> ShowFindInCommitFilesGitGrep { get; } = Setting.Create(AppearanceSettingsPath, nameof(ShowFindInCommitFilesGitGrep), false);
```

Comment:
I prefer to not to call it filter. It filters file in the commit, but the diff tab is by default only the diff, this adds info.
Therefore I propose something with Box instead of Filter.
Similar with the Filter below. Occasionally Filter is used, occasionally FindFiles. Inconsistent.

Reply:
The settings name aligns with all other names for objects and variables. Renaming it to "GitGrep" or "GitGrepBox" will make it inconsistent and disconnected from the objects this setting controls.